### ISSUE TITLE
Italic syntax parsing error.

### ISSUE THREAD

**Issue by nickcheng** [MacDownApp/macdown#195]
Sample below

```
_abc: This variable is similar with `_bcd`
```

should be rendered as:

_abc: This variable is similar with `_bcd`

But in MacDown, it was rendered as:

_abc: This variable is similar with `_ bcd` (There is no space before "bcd")

**Comment by GitBruno** [comment]
Here is a filename [example](http://johnmacfarlane.net/babelmark2/?text=*+ISOcoated_v2_eci.icc) of misparsing:

    * ISOcoated_v2_eci.icc

Should the first underscore should be preceded and the second one followed by a space for parsing?

Why is this on hold?

**Comment by franklinyu** [comment]
@GitBruno Because it only relates to the upstream project, I suppose.

**Comment by GitBruno** [comment]
Right! I see a support for [CommonMark](http://commonmark.org) request in the upstream issues but it's from last year. Would be good, and solves this issue too. I've read a bit more about CommonMark and have a better understanding as to why it is on hold. LOL

**Comment by franklinyu** [comment]
@GitBruno This idea is recently raised also in https://github.com/MacDownApp/macdown/issues/791#issuecomment-296497433. Just saying.

**Comment by utdrmac** [comment]
Very old bug but it is a pertinent issue. +1

Works as expected in CommonMark:

https://spec.commonmark.org/dingus/?text=Testing%20*innodb_file_per_table*%20this%20is%20a%20test
